editor/editor_themes.cpp

```diff
@@ -327,7 +331,11 @@ void editor_register_and_generate_icons(Ref<Theme> p_theme, bool p_dark_theme =
 				}
 
 				const int is_exception = exceptions.has(editor_icons_names[i]);
-				icon = editor_generate_icon(i, !is_exception, EDSCALE, saturation, dark_icon_color_dictionary);
+				if (is_exception) {
+					icon = editor_generate_icon(i, EDSCALE, saturation, dummy_color_map);
```

Comment:
```suggestion
					icon = editor_generate_icon(i, EDSCALE, saturation, HashMap<Color, Color>());
```

Reply:
This was my original code, but it failed the build, unless changing it to `const` in the definition somehow fixes it.

Reply:
It's a temporary variable, so the argument must be `const`.